windows_sg.tf

```diff
@@ -0,0 +1,52 @@
+# Security group for the Windows instances
+resource "aws_security_group" "windows" {
+  provider = aws.provisionassessment
+
+  vpc_id = aws_vpc.assessment.id
+
+  tags = {
+    Name = "Windows"
+  }
+}
+
+# Allow egress to PenTest Portal instances via ports 443 and 8080
+resource "aws_security_group_rule" "windows_egress_to_pentestportal_via_web" {
+  for_each = toset(["443", "8080"])
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.windows.id
+  type                     = "egress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.pentestportal.id
+  from_port                = each.key
+  to_port                  = each.key
+}
+
+# Allow egress to Nessus instances via port 8834 (the default port
+# used by the Nessus web UI)
+resource "aws_security_group_rule" "windows_egress_to_nessus_via_web_ui" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.windows.id
+  type                     = "egress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.nessus.id
+  from_port                = 8834
+  to_port                  = 8834
+}
```

Comment:
We don't want these security group rules in place, for the reasons explained above.